gcloud/bigtable/cluster.py

```diff
@@ -248,3 +250,27 @@ def delete(self):
         # We expect a `._generated.empty_pb2.Empty`
         self._client._cluster_stub.DeleteCluster(
             request_pb, self._client.timeout_seconds)
+
+    def list_tables(self):
+        """List the tables in this cluster.
+
+        :rtype: list of :class:`Table <gcloud.bigtable.table.Table>`
+        :returns: The list of tables owned by the cluster.
+        :raises: :class:`ValueError <exceptions.ValueError>` if one of the
+                 returned tables has a name that is not of the expected format.
+        """
+        request_pb = table_messages_pb2.ListTablesRequest(name=self.name)
+        # We expect a `table_messages_pb2.ListTablesResponse`
+        table_list_pb = self._client._table_stub.ListTables(
+            request_pb, self._client.timeout_seconds)
+
+        result = []
+        for table_pb in table_list_pb.tables:
+            before, table_id = table_pb.name.split(
+                self.name + '/tables/', 1)
+            if before != '':
+                raise ValueError('Table name %s not of expected format' % (
+                    table_pb.name,))
```

Comment:
The expected format is `{cluster_name}/tables/{table_name}`, which amounts to `projects/../zones/../clusters/../tables/{table_id}`.

The reason I didn't use a regex is because we already have `cluster_name` easy to access.